cli/lambda-node-version-update.md

```diff
@@ -1,4 +1,57 @@
 # Node Version Update
+
+## Node.js 8.10 to Node.js 10.x
+According to AWS Lambda [Runtime Support Policy](https://docs.aws.amazon.com/lambda/latest/dg/runtime-support-policy.html), AWS Lambda deprecates Node.js 
+runtime Node.js 8.10 on January 6th, 2020. 
+
+The Amplify CLI code base has been updated to reflect this change. Amplify CLI replaces Node.js 8.10 with Node.js 10 in the Lambda functions that it creates for you. If you use Amplify CLI version 4.10.0 and above to create new aws resources, this does not concern you.
+
+However, if you have used previous versions of the Amplify CLI to create aws resources in the following categories, 
+you will need to manually update your project to avoid Node.js runtime issues with AWS Lambda. 
+- auth
```

Comment:
Add `functions` to this list
Add a functions section as well to basically explain the change in import/export behavior for files co-located in the same directory